modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,151 @@
+    # Performance Addon Operator
+    - fileName: PaoSubscriptionNS.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-ns"
+    - fileName: PaoSubscriptionCatalogSource.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-cat-source"
+      spec:
+        image: <image URL here>
+    - fileName: PaoSubscription.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-sub"
+    #These are custom local polices that come from the source-crs directory in the git repo
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
+      remediationAction: inform
+      policyName: "group-dev-elasticsearch-ns"
+    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
+      remediationAction: inform
+      policyName: "group-dev-elasticsearch-operator-group"
+    #Custom Resources
+    - fileName: custom/apiserver-config.yaml
+      remediationAction: inform
+      policyName: "group-dev-apiserver-config"
+    - fileName: custom/disable-nic-lldp.yaml
+      remediationAction: inform
+      policyName: "group-dev-disable-nic-lldp"
 ----
+<1> Set `fileName` to include the custom CR subfolder from the `/source-crs` parent.
+
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP Argo CD policies application.
 
-. Push the built container image to your disconnected registry, for example:
+. Update the `ClusterGroupUpgrade` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`, as shown in the following example:
 +
-[source,terminal,subs="attributes+"]
+[source,yaml]
 ----
-$ podman push localhost/ztp-site-generate-rhel8-custom:v{product-version}-custom-1 registry.example.com:5000/ztp-site-generate-rhel8-custom:v{product-version}-custom-1
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: custom-source-cr
+  namespace: ztp-clusters
+spec:
+  managedPolicies:
+    - group-dev-config-policy
+  enable: true
+  clusters:
+  - cluster1
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
 ----
 
-. Patch the Argo CD instance on the hub cluster to point to the newly built container image:
+. Apply the updated `ClusterGroupUpgrade` CR by running the following command:
+[source, terminal]
 +
-[source,terminal,subs="attributes+"]
 ----
-$ oc patch -n openshift-gitops argocd openshift-gitops --type=json -p '[{"op": "replace", "path":"/spec/repo/initContainers/0/image", "value": "registry.example.com:5000/ztp-site-generate-rhel8-custom:v{product-version}-custom-1"} ]'
+$ oc apply -f cgu-test.yaml
 ----
-+
-When the Argo CD instance is patched, the `openshift-gitops-repo-server` pod automatically restarts.
 
 .Verification
 
-. Verify that the new `openshift-gitops-repo-server` pod has completed initialization and that the previous repo pod is terminated:
+. Check that the updates have succeeded by running the following command:
```

Comment:
[A single-step procedure should use an unnumbered list item](https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures): 

```suggestion
* Check that the updates have succeeded by running the following command:
```